### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by eldruin**
The changes seemed straight-forward. I have not tested this in hardware, though.

### CODE REVIEW

**Comment by sajattack** [comment]
Thanks, I'll try to find some time to test and merge this on the weekend, remind me if I forget.

**Comment by eldruin** [comment]
Hi there, here is the reminder :)

File: Cargo.toml [hunk 1]
```diff
@@ -11,19 +11,20 @@ keywords = ["no_std", "embedded", "bitbang", "embedded-hal", "hal"]
...
-cortex-m = "0.6.2"
-cortex-m-rt = "0.6.12"
+cortex-m = "0.7"
+cortex-m-rt = "0.7"
 panic-halt = "0.2.0"
-eeprom24x = "0.3.0"
-lm75 = "0.1"
+eeprom24x = "0.5.0"
+lm75 = "0.2"
+fugit = "0.3"
```
Comment (sajattack) [inline Cargo.toml]: why is this needed? (fugit)

**Comment by sajattack** [comment]
Looks good
SDS00001

Just let me know if that fugit dep is needed, and if not, please remove it, then I'll merge.

And sorry for the delay.

File: Cargo.toml (hunk 1 above)
Comment (eldruin) [reply Cargo.toml]: Hmm, I had to add it for some reason but now without it seems to build fine as well so I took it out again.
